manual_user_install.sh

```diff
@@ -58,6 +57,7 @@ function install_core (){
     sed -i '/\@deprecated/d' $HOME/.local/lib/python3.9/site-packages/ovos_utils/fingerprinting.py
     sed -i '/\@deprecated/d' $HOME/.local/lib/python3.9/site-packages/ovos_utils/configuration.py
     sed -i '/\@deprecated/d' $HOME/.local/lib/python3.9/site-packages/ovos_utils/ovos_service_api.py
+    sed -i '/\@deprecated/d' $HOME/.local/lib/python3.9/site-packages/ovos_utils/signal.py
```

Comment:
whats the purpose of these sed commands?

Reply:
Day before yesterday logs were bogged down with deprecated messages from these four python scripts.  Deprecated functions had been annotated with "@deprecated" that appears to generate an entry telling developers about functions that call deprecated functions.  Runtime more than doubled and timeouts were getting triggered (remember I'm working on a Pi 3).  On a faster system you might not even notice, on the slow system I wasn't even able to test.  I can see where a developer would want to see those messages, but not sure why a developer would be doing their work on a Pi 3.  Let me know if the @deprecated tags need to stay in the code.

Reply:
understood, lots of PRs today getting rid of those so maybe this is no longer needed

PRs or issue in relevant repos welcome whenever those logs show up, they indicate precisely the line that needs changing

the goal of these is to inform skill developers coming from mycroft mostly, the usages in ovos components should be fixed before 0.0.8 release and you won't have too many of these by default

Reply:
I'm going to comment them out for now with an explanation for when to un-comment them.